shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumTransferLeaderCommand.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.journal;
+
+import alluxio.cli.fsadmin.command.AbstractFsAdminCommand;
+import alluxio.cli.fsadmin.command.Context;
+import alluxio.client.journal.JournalMasterClient;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.ExceptionMessage;
+import alluxio.exception.status.InvalidArgumentException;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+
+/**
+ * Command for transferring the leadership to another master within a quorum.
+ */
+public class QuorumTransferLeaderCommand extends AbstractFsAdminCommand {
```

Comment:
Let's be consistent with the new naming within this level. (TransferLeader -> Elect)